src/components/Dashboard/EditAccount.js

```diff
@@ -0,0 +1,96 @@
+import React from 'react'
+import { Link, NavLink } from 'react-router-dom'
+import SignOut from '../../components/Auth/SignOut'
+import { connect } from 'react-redux'
+
+import axiosWithAuth from '../../Helpers/axiosWithAuth'
+
+import './Dashboard.scss'
+
+function EditAccount({ user }) {
+	const avatar =
+		user.avatar || 'http://s3.amazonaws.com/37assets/svn/765-default-avatar.png'
+
+	const handleSubmit = e => {
+		// e.preventDefault()
+
+		const data = {
+			username: e.target.username.value,
+			email: e.target.email.value
+		}
```

Comment:
Try using `username: e.target.username.value || user.username`

This way if the input field is empty, it will use the user's current credentials instead of erasing them.